includes/admin/class-wc-admin-post-types.php

```diff
@@ -28,10 +28,11 @@ class WC_Admin_Post_Types {
 	 * Constructor.
 	 */
 	public function __construct() {
-		include_once dirname( __FILE__ ) . '/class-wc-admin-meta-boxes.php';
+		$dir = __DIR__ . DIRECTORY_SEPARATOR;
```

Comment:
To be honest we can avoid using `DIRECTORY_SEPARATOR`, since in some instances yet there's hard coded `/`, also it got replaces on Windows, we shouldn't worry about it.

Reply:
Thanks @claudiosanches, changes made in 67a178f